extendeddatacrossword_test.go

```diff
@@ -110,3 +111,44 @@ func TestRepairExtendedDataSquare(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkRepair(b *testing.B) {
+	for _codecType := range codecs {
+		bufferSize := 64
+		ones := bytes.Repeat([]byte{1}, bufferSize)
+		twos := bytes.Repeat([]byte{2}, bufferSize)
+		threes := bytes.Repeat([]byte{3}, bufferSize)
+		fours := bytes.Repeat([]byte{4}, bufferSize)
+
+		original, err := ComputeExtendedDataSquare([][]byte{
+			ones, twos,
+			threes, fours,
+		}, _codecType, NewDefaultTree)
+		if err != nil {
+			panic(err)
+		}
+
+		flattened := original.flattened()
+		flattened[0], flattened[2], flattened[3] = nil, nil, nil
+		flattened[4], flattened[5], flattened[6], flattened[7] = nil, nil, nil, nil
+		flattened[8], flattened[9], flattened[10] = nil, nil, nil
+		flattened[12], flattened[13] = nil, nil
```

Comment:
Lose data randomly as in other places, too. (e.g. see how @evan-forbes used `generateMissingData`).